test/acceptance/features/steps/openshift.py

```diff
@@ -373,3 +373,13 @@ def get_deployment_with_intermediate_secret_of_given_pattern(self, intermediate_
             else:
                 print(f"No deployment that matches {deployment_name_pattern} found.\n")
         return None
+
+    def delete_service_binding(self, sb_name):
+        (output, exit_code) = self.cmd.run("oc delete ServiceBinding " + sb_name)
```

Comment:
the current command does not really wait for deletion to happen - please add `-wait` and `--timeout` flags to make it more reliable.

Reply:
done